### ISSUE TITLE
Review Question

### ISSUE THREAD

**Issue by VincentCheng51** [SENECA-DSA555/dsa555-s16#14]
quick sort is often the fastest sorting algorythm, can quick sort in specific case(s) run as slow as bubble sort? if not, explain how bubble sort is slower than quick sort and if so give an example of which case

**Comment by cathyatseneca** [comment]
Good question!

**Comment by donaldtn** [comment]
Yes, quick sort can run as slow as bubble sort if the data set is small.

**Comment by cathyatseneca** [comment]
this is the reason @donaldtn.  That isn't the reason for the case of quicksort being O(n^2).  n can be any number and usually big because big O refers to a growth rate. if you are dealing with a fixed value, then the data set never grows and therefore the number of step never changes so there is no growth.

**Comment by donaldtn** [comment]
Let me try again...

Quick sort will be as slow as bubble if the pivot happens to be the largest or smallest element in the data set.

**Comment by cathyatseneca** [comment]
how often does that need to happen?  ... elaborate more.
